sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - Custom form recognition without labels can now handle multipaged forms.
 - `RecognizedForm.Pages` now only contains pages whose numbers are within `RecognizedForm.PageRange`.
 - `FieldText.TextContent` cannot be `null` anymore, and it will be empty when no element is returned from the service.
+- `FormRecognizerClient.StartRecognizeCustomFormsFromUri` now works with URIs that contain blank spaces, encoded or not.
```

Comment:
I was looking at the [keyvault](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/CHANGELOG.md) changelog and noticed that they put the issue number next to the fixes.
After further look, found this in the guidance: https://github.com/Azure/azure-sdk/blob/2ef0280933f7e6859360c27912bc75e5a64b0bba/docs/policies/releases.md#changelog-guidance

Could you update this section?

Reply:
Updated the first and last items. The second and third ones didn't have issues associated with them.